contrib/spring/src/main/java/io/opencensus/contrib/spring/instrument/web/client/TracingAsyncClientHttpRequestInterceptor.java

```diff
@@ -57,22 +58,36 @@ public void put(HttpRequest carrier, String key, String value) {
   /**
    * Create an instance of {@code TracingAsyncClientHttpRequestInterceptor}.
    *
+   * @param extractor {@link HttpExtractor} to extract request and response specific attributes. If
+   *     it is null then default extractor is used.
+   * @param propagator {@link TextFormat} to propagate trace context to remote peer. If it is null
+   *     then default propagator (TraceContextFormat) is used.
    * @return {@code TracingAsyncClientHttpRequestInterceptor}
    * @since 0.23.0
    */
-  public static TracingAsyncClientHttpRequestInterceptor create() {
-    return new TracingAsyncClientHttpRequestInterceptor();
+  public static TracingAsyncClientHttpRequestInterceptor create(
+      /* @Nullable */ TextFormat propagator,
+      /* @Nullable */ HttpExtractor<HttpRequest, ClientHttpResponse> extractor) {
+    return new TracingAsyncClientHttpRequestInterceptor(propagator, extractor);
   }
 
-  TracingAsyncClientHttpRequestInterceptor() {
+  TracingAsyncClientHttpRequestInterceptor(
+      /* @Nullable */ TextFormat propagator,
```

Comment:
```suggestion
      @javax.annotations.Nullable TextFormat propagator,
```

Same for all others.